src/main/java/study/step2/StringCalculator.java

```diff
@@ -0,0 +1,54 @@
+package study.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    private static final String SEPARATOR = ",|:";
+    private static final String CUSTOM_SEPARATOR = "//(.)\n(.*)";
+
+    public static int calculate(String input) {
+        if (checkNullOrEmpty(input)) {
+            return 0;
+        }
+
+        String[] numbers = split(input);
+
+        int sum = sum(numbers);
+        return sum;
```

Comment:
```suggestion
        return sum(split(input));
```
불필요한 로컬 변수 사용하지 않으면 어떨까?